From 69ce92ad59f8bb5b00fc20cb5d6dff8d43fc7da9 Mon Sep 17 00:00:00 2001
From: Kalle Jokiniemi <kalle.jokiniemi@jolla.com>
Date: Thu, 28 Aug 2014 19:11:24 +0300
Subject: [PATCH 1/2] [make] Fix compilation by increasing optimization to -O2

Building with mer toolchaing the -O1 option fails to build.
So increase optimization to -O2 which does compile.

Signed-off-by: Kalle Jokiniemi <kalle.jokiniemi@jolla.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index e721e99..f8b928c 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@ CC = gcc
 LN = ln
 AR = ar
 AM_CFLAGS = -Wall -D_FILE_OFFSET_BITS=64 -DBTRFS_FLAT_INCLUDES -fno-strict-aliasing -fPIC
-CFLAGS = -g -O1 -fno-strict-aliasing
+CFLAGS = -g -O2 -fno-strict-aliasing
 objects = ctree.o disk-io.o radix-tree.o extent-tree.o print-tree.o \
 	  root-tree.o dir-item.o file-item.o inode-item.o inode-map.o \
 	  extent-cache.o extent_io.o volumes.o utils.o repair.o \
-- 
1.8.3.2

